### ISSUE TITLE
"Security framework of XStream not initialized, XStream is probably vulnerable."

### ISSUE THREAD

**Issue by toyg** [TheBoegl/gradle-launch4j#82]
I keep getting a warning related to XStream in my builds - why is it there and how do I turn it off? I don't use XStream at all myself... 
```
11:53:34 PM: Executing task 'createAllExecutables'...

> Task :compileJava UP-TO-DATE
> Task :processResources NO-SOURCE
> Task :classes UP-TO-DATE
> Task :shadowJar

> Task :createExe
Security framework of XStream not initialized, XStream is probably vulnerable.

> Task :createAllExecutables

BUILD SUCCESSFUL in 4s
3 actionable tasks: 2 executed, 1 up-to-date
11:53:38 PM: Task execution finished 'createAllExecutables'.
```

**Comment by bitbythecron** [comment]
FWIW I had the same issue and specifying **both** min and max JRE version solved it for some reason 🤷‍♂️

**Comment by TheBoegl** [comment]
This issue should be fixed with the launch4j 2.14 used in version 2.5.0 of this plugin
